### ISSUE TITLE
"Shortcut" linked shared files/folders don't show up

### ISSUE THREAD

**Issue by jacobmischka** [jstaf/onedriver#157]
In the OneDrive web client you can link a shared folder to your "My files" directory by pressing "Add shortcut to My files".

![image](https://user-images.githubusercontent.com/3939997/129210972-36800157-f793-42ae-b172-ad4bb602d1bd.png)

I'm not sure how this works internally, so it might not be possible to support, but it would be nice if these showed up in the mounted OneDriver directory as well, as they currently do not seem to.

![image](https://user-images.githubusercontent.com/3939997/129210999-7a61f5c1-4059-49fe-a8af-c661babbc393.png)

**Comment by jstaf** [comment]
Yeah, need to support shared files first, but should doable once that's setup.

**Comment by abraunegg** [comment]
@jstaf , @jacobmischka 

This will never be possible. The OneDrive API does not present the 'shortcut' data - see: https://github.com/OneDrive/onedrive-api-docs/issues/1427

**Comment by jstaf** [comment]
Oh, that's fun. I love gaps in the Microsoft APIs like this. Thanks for the heads up!

**Comment by jacobmischka** [comment]
Dang, thanks anyway for this great tool!

**Comment by magicmatt007** [comment]
Meanwhile, was anything changed on Microsoft's API, which would allow making this feature available now?

**Comment by mkobel** [comment]
There is a workaround:

Add the header `Prefer: Include-Feature=AddToOneDrive` to the requests.

I found the hint for this workaround here:
- https://github.com/rclone/rclone/issues/5428#issuecomment-1755151325
- https://github.com/OneDrive/onedrive-api-docs/issues/1674#issuecomment-1494727537

It worked for me with rclone (` --header 'Prefer: Include-Feature=AddToOneDrive'`) and one our applications where we use the Java SDK


Edit:
I added the header in `fs/graph/graph.go`. The shortcuts appeared in the file browser as files with size 0. I therefore assume the handling of the Shortcut-data has to be implemented too.
